cumulus/parachains/runtimes/collectives/collectives-westend/src/weights/xcm/mod.rs

```diff
@@ -262,7 +262,8 @@ impl<Call> XcmWeightInfo<Call> for CollectivesWestendXcmWeight<Call> {
 		XcmGeneric::<Runtime>::clear_topic()
 	}
 	fn alias_origin(_: &Location) -> Weight {
-		XcmGeneric::<Runtime>::alias_origin()
```

Comment:
This should stay like this so we can use the new `preserve_origin` feature of `InitiateTransfer` in XCMv5